### DISCUSSION TITLE
Scoop appreciation thread

### DISCUSSION THREAD

**Discussion by keenbowl3009** [ScoopInstaller/Scoop#4517] [General]
Hello community, 

I would like to know about ways in which we can support the scoop community and their efforts. 

We're seeing aggressive marketing of winget by microsoft, and as we all know, winget has absolutely no philosophy. 

I would like to know what is the philosophy of scoop, and how much in agreement with it the scoop community stands!

Here is a problem that shows the lack of philosophy: 

https://github.com/microsoft/winget-cli/issues/1711

The same lack of philosophy does not occur with scoop packages because they are as easy to get rid of than they are of installing!

**Comment by keenbowl3009** [comment]
Another huge problem of winget's lack of philosophy and microsoft's inferior implementation of products, is that to this day, a feature that is marked as fixed, is still broken, which is the ability to uninstall software from a windows system. 

https://github.com/microsoft/winget-cli/issues/121

**Comment by jthoward64** [comment]
If you want to have a discussion make a discussion (big button at the top if the screen). 
Issues are for bugs, feature requests, etc.

**Comment by JanPokorny** [comment]
Scoop is made for unzipping CLI tools -- ideal for Git, Python, ffmpeg...
WinGet is made for running installers -- ideal for Steam, IDEs, browsers...

For ideal workflow, it is beneficial to use both.

**Comment by keenbowl3009** [comment]
"WinGet is made for running installers -- ideal for Steam, IDEs, browsers"

That is a fantastic description. Emphasis on running installers - winget will not allow you to properly uninstall most things that you have acquired with it.

**Comment by JanPokorny** [comment]
@keenbowl3009 Not my experience at all. If the installer properly registers an uninstaller, winget will run it.

**Comment by rashil2000** [comment]
The functionalities of Winget and Scoop are largely orthogonal - they serve different purposes and do them well. Winget allows you to cleanly uninstall inbox UWP apps (looking at you Cortana) - something that was nearly impossible manually. I haven't faced any uninstallation issues (yet) with Winget.

Here's a previous discussion related to this - https://github.com/ScoopInstaller/Scoop/issues/3992 - with comments from the creator of Scoop itself.

**Reply by EqualDust** [reply]
Try uninstalling "calibre.calibre.portable": https://github.com/microsoft/winget-cli/issues/1750

**Comment by EqualDust** [comment]
Obligatory reference to winget uninstaller being broken for yet another package. And the msft team marked the issue as done! Qwality! https://github.com/microsoft/winget-cli/issues/121

**Reply by JanPokorny** [reply]
Are you the same guy again?

**Reply by rashil2000** [reply]
I don't understand the hate, really. If you don't like a product, just... don't use it!?

Winget is barely a year old, and nobody is forcing you to use it.

**Comment by EqualDust** [comment]
Winget is the default windows package manager. It comes bundled with windows. Microsoft encourages windows users to use the tool: https://docs.microsoft.com/en-us/windows/package-manager/winget/

**Reply by rashil2000** [reply]
I don't see any reference in the docs that _encourage_ you to use it. It's a doc for those who are looking for that doc.

Besides, Encouraged != Forced

**Comment by keenbowl3009** [comment]
Glad to know then that Microsoft does not encourage users to use winget, the default windows package manager. 😂

**Reply by JanPokorny** [reply]
I see you switched back to your old account now. You surely use a lot of energy on bashing winget! Why?

You made me self-reflect and realize that my effort on you is wasted. From now, I will stop commenting and only use the "thumbs down" emoji.

**Reply by rashil2000** [reply]
@keenbowl3009 @EqualDust @keentext @BentramLawrence @ShanConn, how many more accounts will you make?

If you want to flame about something, write a blog and publish it somewhere. Don't bug people here with such non-constructive comments. You will get blocked.

**Comment by chawyehsu** [comment]
STOP trolling please.
